staging/vhost-device-sound/src/stream.rs

```diff
@@ -11,6 +11,8 @@ use crate::{virtio_sound::*, Direction, IOMessage, SUPPORTED_FORMATS, SUPPORTED_
 /// Stream errors.
 #[derive(Debug, ThisError, PartialEq, Eq)]
 pub enum Error {
+    #[error("Guest driver request in an invalid stream state {0}")]
+    InvalidState(PCMState),
```

Comment:
Maybe codify what request led to this error as well?

Reply:
Do you think something like this would be OK?
```diff
-    #[error("Guest driver request in an invalid stream state {0}")]
-    InvalidState(PCMState),
+    #[error("Guest driver request {0} in an invalid stream state {1}")]
+    InvalidState(&'static str, PCMState),
```
And then, to use like this:
```diff
             return Err(Error::Stream(crate::stream::Error::InvalidState(
+                "write",
                 self.streams.read().unwrap()[stream_id as usize].state,
             )));
```

Reply:
yeah!